src/vaadin-overlay.html

```diff
@@ -706,7 +708,7 @@
 
         // Disable pointer events in other attached overlays
         OverlayElement.__attachedInstances.forEach(el => {
-          if (el !== this) {
+          if (el !== this && el._isAttached) {
```

Comment:
Could we avoid adding an extra flag with:
`if (el !== this && !el.hasAttribute('opening') && !el.hasAttribute('closing')) {`

Reply:
Done.